src/index.html

```diff
@@ -11,12 +11,53 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <img src="./images/logo.png" alt="logo Moyo">
+      </div>
+      </a>
```

Comment:
There is an unmatched closing `</a>` tag here. It seems to be misplaced as there is no corresponding opening `<a>` tag. Please remove or correct this to maintain proper HTML structure.